src/libraries/System.IO.MemoryMappedFiles/tests/MemoryMappedFile.CreateFromFile.Tests.cs

```diff
@@ -36,7 +36,17 @@ public void InvalidArguments_Path()
         public void InvalidArguments_FileStream()
         {
             // null is an invalid stream
-            AssertExtensions.Throws<ArgumentNullException>("fileStream", () => MemoryMappedFile.CreateFromFile(null, CreateUniqueMapName(), 4096, MemoryMappedFileAccess.Read, HandleInheritability.None, true));
+            AssertExtensions.Throws<ArgumentNullException>("fileStream", () => MemoryMappedFile.CreateFromFile((FileStream)null, CreateUniqueMapName(), 4096, MemoryMappedFileAccess.Read, HandleInheritability.None, true));
+        }
+
+        /// <summary>
+        /// Tests invalid arguments to the CreateFromFile fileHandle parameter
+        /// </summary>
+        [Fact]
+        public void InvalidArguments_SafeFileHandle()
+        {
+            // null is an invalid stream
```

Comment:
nit: minor copy-paste error

```suggestion
            // null is an invalid handle
```